neo.UnitTests/TestUtils.cs

```diff
@@ -107,6 +107,28 @@ public static Transaction CreateRandomHashTransaction()
             }
 
             return newObj;
+        }
+		
+		public static void DeleteFile(string file)
```

Comment:
indentation

Reply:
please run `dotnet format` if you have it available.

Reply:
Good tool，thanks.